scripts/install/install_macos.sh

```diff
@@ -566,22 +572,37 @@ install_package()
   decrease_indent
 }
 
-# create_manager_yml creates the manager.yml at the specified path, containing opamp information.
-create_manager_yml()
+create_supervisor_config()
 {
-  manager_yml_path="$1"
-  if [ ! -f "$manager_yml_path" ]; then
-    info "Creating manager yaml..."
+  supervisor_yml_path="$1"
+  if [ ! -f "$supervisor_yml_path" ]; then
+    info "Creating supervisor config..."
 
     # Note here: We create the file and change permissions of the file here BEFORE writing info to it
     # We do this because the file may contain a secret key, so we want 0 window when the
     # file is readable by anyone other than root
-    command printf '' >> "$manager_yml_path"
-    chmod 0600 "$manager_yml_path"
-
-    command printf 'endpoint: "%s"\n' "$OPAMP_ENDPOINT" > "$manager_yml_path"
-    [ -n "$OPAMP_LABELS" ] && command printf 'labels: "%s"\n' "$OPAMP_LABELS" >> "$manager_yml_path"
-    [ -n "$OPAMP_SECRET_KEY" ] && command printf 'secret_key: "%s"\n' "$OPAMP_SECRET_KEY" >> "$manager_yml_path"
+    command printf '' >> "$supervisor_yml_path"
+    chmod 0600 "$supervisor_yml_path"
+
+    command printf 'server:\n' > "$supervisor_yml_path"
+    command printf '  endpoint: "%s"\n' "$OPAMP_ENDPOINT" >> "$supervisor_yml_path"
+    command printf '  headers:\n' >> "$supervisor_yml_path"
+    [ -n "$OPAMP_SECRET_KEY" ] && command printf '    Authorization: "Secret-Key %s"\n' "$OPAMP_SECRET_KEY" >> "$supervisor_yml_path"
+    # [ -n "$OPAMP_LABELS" ] && command printf '    X-Bindplane-Attribute: "%s"\n' "$OPAMP_LABELS" >> "$supervisor_yml_path"
+    command printf '  tls:\n' >> "$supervisor_yml_path"
+    command printf '    insecure: true\n' >> "$supervisor_yml_path"
+    command printf '    insecure_skip_verify: true\n' >> "$supervisor_yml_path"
+    command printf 'capabilities:\n' >> "$supervisor_yml_path"
+    command printf '  accepts_remote_config: true\n' >> "$supervisor_yml_path"
+    command printf '  reports_remote_config: true\n' >> "$supervisor_yml_path"
+    command printf 'agent:\n' >> "$supervisor_yml_path"
+    # TODO(dakota): Add logging config option when supervisor suppports it
+    command printf '  executable: "%s"\n' "$INSTALL_DIR/observiq-otel-collector" >> "$supervisor_yml_path"
+    command printf '  description:\n' >> "$supervisor_yml_path"
+    command printf '    non_identifying_attributes:\n' >> "$supervisor_yml_path"
+    [ -n "$OPAMP_LABELS" ] && command printf '      service.labels: "%s"\n' "$OPAMP_LABELS" >> "$supervisor_yml_path"
+    command printf 'storage:\n' >> "$supervisor_yml_path"
+    command printf '  directory: "%s"\n' "$INSTALL_DIR/storage" >> "$supervisor_yml_path"
```

Comment:
Let's have this storage directory by something other than `storage`; My thinking is I don't want it to potentially conflict with other storage for OTel components (e.g. filestorage extension).

Maybe we could just have this go to a separate `supervisor_storage` directory?

Reply:
sounds good